scanner/Makefile

```diff
@@ -81,21 +100,21 @@ image-prefix := scanner-v4
 .PHONY: images
 images: scanner-image db-image
 
-scanner-go-build: deps
+image/scanner/bin/scanner: bin/scanner
 	@echo "+ $@"
-	$(BUILD_CMD)
+	$(SILENT)cp $< $@
 
 .PHONY: scanner-go-build-local
 scanner-go-build-local:
```

Comment:
Does this name still make sense with the new changes?

Reply:
Generally, yes. Let me know if you have some specific in mind.